api/src/main/java/com/spotify/spydra/api/DataprocAPI.java

```diff
@@ -113,7 +114,7 @@ public boolean updateMasterMetadata(SpydraArgument arguments, String key, String
     return success;
   }
 
-  public Collection<Cluster> listClusters(SpydraArgument arguments) throws IOException {
-    return gcloud.listClusters(arguments.cluster.getOptions().get("project"), arguments.getRegion());
+  public Collection<Cluster> listClusters(SpydraArgument arguments, Map<String, String> filters) throws IOException {
```

Comment:
I wonder how much of the underlying `gcloud` implementation for `dataproc` we are leaking through here. Not something I worry about overly much though.